draft-quic-atsss-reqs.mkd

```diff
@@ -346,10 +346,6 @@ Nevertheless, blindly distributing packets among existing paths may lead to comp
 
 - Frequent connection migration if both access networks are used to forward data; each with a distinct source IP address.
 
-The deployment of MPTCP may bring high complexity and high cost, e.g. MPTCP client requires native support in mobile phone as well as chip, which leads to difficulty to wide-scale MPTCP deployment. MPQUIC is implemented in application layer, which can bring more convenience in deployment.
```

Comment:
Could we write something like:
- MPTCP typically relies on specific support inside the operating system of the UE. It is not yet enabled by default by all UE vendors but MPTCP support is improving. MPQUIC is implemented in the application layer.

Reply:
This is much more better. Thanks.